scrape_all_prs.py

```diff
@@ -104,8 +107,10 @@ def collect_pr_metadata():
                 pr.updated_at, 
                 "merged" if pr.merged else "closed", 
                 pr.changed_files, 
-                len([comment for comment in pr.get_comments() if comment.user.type != "Bot"]), 
+                sum([file.changes for file in pr.get_files()]),
+                len([comment for comment in pr.get_comments() + pr.get_issue_comments() if comment.user.type != "Bot"]), 
```

Comment:
Total comments includes pull request review comments (on unified diff) and issue comments (not on diff)